homeassistant/components/ring/__init__.py

```diff
@@ -50,4 +66,23 @@ def setup(hass, config):
             title=NOTIFICATION_TITLE,
             notification_id=NOTIFICATION_ID)
         return False
+
+    def hub_refresh(event_time):
```

Comment:
When this is the service handler, the argument will not be the service time. Wrapping this function in a specific service handler.

Reply:
changed, I think if I understood the issue correctly!